src/Syntax/TextmateTokenizerJob.re

```diff
@@ -100,6 +101,20 @@ let onBufferUpdate = (bufferUpdate: BufferUpdate.t, lines, v: t) => {
   Job.map(f, v);
 };
 
+let _hexToColor = Hashtbl.create(128);
+
+let hexToColor = (hex: string) => {
+  switch (Hashtbl.find_opt(_hexToColor, hex)) {
+  | None =>
+    let color = Revery.Color.hex(hex);
+    Hashtbl.add(_hexToColor, hex, color);
+    color;
+  | Some(color) => color
+  };
+};
+
+exception NoWhitespaceException;
```

Comment:
You can define local exceptions inside functions now (since 4.08 maybe?)

Reply:
Oh, cool! Didn't know that. I factored this out to use `StringEx.forAll` instead, but good to know for the future.